test/unit/test_utils.py

```diff
@@ -0,0 +1,130 @@
+"""
+test_utils.py
+Date: 09/07/2020
+Author: Mihai Coșleț
+Email: coslet.mihai@gmail.com
+"""
+
+from pathlib import Path
+
+import pytest
+
+from rdf_differ import defaults
+from rdf_differ.utils import dir_exists, file_exists, dir_is_empty, get_envs
+
+
+def helper_endpoint_mock(monkeypatch):
+    monkeypatch.setenv('ENDPOINT', 'http://test.point')
```

Comment:
And again, as I mentioned before, having the env patching in a fixture is making the test hard to read as you have to go and look to the fixture and check what env var is patched. I'd say better to move this into the test.